misc/xml-to-record-converter/src/main/java/io/ballerina/xmltorecordconverter/XMLToRecordRequest.java

```diff
@@ -0,0 +1,72 @@
+/*
+ *  Copyright (c) 2023, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package io.ballerina.xmltorecordconverter;
+
+/**
+ * Request format for XMLToRecord endpoint.
+ *
+ * @since 2.0.0
+ */
+public class XMLToRecordRequest {
+
+    private String xmlValue;
+    private boolean isRecordTypeDesc;
+    private boolean isClosed;
+    private boolean forceFormatRecordFields;
```

Comment:
are all of these parameters applicable for the xml conversions as well?

Reply:
yes, at the moment, not all are implemented, but yes